platforms/generic-pi/boot-config-loader/pkg.yaml

```diff
@@ -0,0 +1,17 @@
+name: rpi-boot-config-loader
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  - image: "ghcr.io/ogkevin/talos-ext-generic-pi:{{.VERSION}}"
```

Comment:
quick nits:

we don't refer an external built tool, we download source and build since that means any security updates gets passed down the toolchain.

Secondly looking at the source, there seems to be the block devices are hardcoded as `/dev/mmcblk` etc, it would be easier to use the talos `go-blockdevice` library to find the parition with the `BOOT` label

Reply:
RE your second point, I was planning on doing that indeed in a later stage. Not sure exactly how. Initial idea was to read the Talos config and get the install disk. Your suggestion also works. 

Is this feature required to get this MR merged? I can create an issue on the repo as well to track this progress if that helps.